src/config.ts

```
@@ -146,37 +140,30 @@ export async function createViemClient(
+    record.recordRequest();
 }
 
 /**
  * Keeps record of http fetch responses for a http viem client
  */
-export function onFetchResponse(response: Response, rpcRecords: Record<string, RpcRecord>) {
+export function onFetchResponse(response: Response, rpcState: RpcState) {
     let url = response.url;
     if (!response.url.endsWith("/")) url = url + "/";
-    let record = rpcRecords[url];
+    let record = rpcState.metrics[url];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)